src/state.py

```diff
@@ -67,8 +143,24 @@ def from_charm(
                 else "[]"
             )
             certificates = json.loads(certificates_json)
+            args = {key.replace("-", "_"): value for key, value in charm.config.items()}
+            valid_config = WazuhConfig(**args)  # type: ignore
```

Comment:
What is the reason for `type: ignore` here please?

Reply:
Added a comment with the explanation